1_basics.ipynb

```diff
@@ -143,23 +256,75 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "1.  Start the simulator environment by entering \"docker-compose up\" command.\n",
```

Comment:
"1.  Start the simulator environment by entering **the** \"docker-compose up\" command.\n",